doc/gettingstarted.md

```diff
@@ -21,32 +23,31 @@ In this guide we will get everything up and running on your local machine (on `l
 
 ## Dependencies 
 
-#### Certificates
+### CLI and Stack
+
+Get the latest packages or binaries for your operating system from the [releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on GitHub.
+
+### Certificates
+
 By default, the Stack requires a `cert.pem` and `key.pem`, in order to to serve content over TLS.
 
-+ To generate testing certificates, you can use the following command. This requires a [Go environment setup](../DEVELOPMENT.md#development-environment).
++ To generate self-signed certificates, you can use the following command. This requires a [Go environment setup](../DEVELOPMENT.md#development-environment).
 
-    ```bash
-    go run $(go env GOROOT)/src/crypto/tls/generate_cert.go -ca -host localhost && chmod 0444 ./key.pem
-    ```
-    
-+ For production certificates we recommend [LetsEncrypt](https://letsencrypt.org/getting-started/).
-    
-<a name="configuration"/>
+```bash
+go run $(go env GOROOT)/src/crypto/tls/generate_cert.go -ca -host localhost && chmod 0444 ./key.pem
+```
 
-## Configuration 
+Keep in mind that self-signed certificates are not trusted by browsers and operating systems, and as such they will return a warning regarding this matter. In order to avoid such issues, we recommend [Let's Encrypt](https://letsencrypt.org/getting-started/).
 
-The Stack can be started without passing any [configuration](config.md). We however recommend paying attention to the following parameters:
+<a name="configuration"/>
 
-+ `TTN_LW_COOKIE_HASHKEY` is a 32 or 64 bytes long parameter, and `TTN_LW_COOKIE_BLOCKKEY` is a 32 bytes long 
-parameter. Both are used for cookie secrets. They should be passed in a hexadecimal string form. If no value is passed, a random value will be generated at startup.
+## Configuration 
```

Comment:
Does `## <a name="configuration">Configuration</a>` not work?

Reply:
The rendering no longer works correctly when Markdown is combined with HTML.

## <a name="configuration">Configuration</a>